docs/authoring/characters.md

```diff
@@ -1,250 +1,436 @@
+```
+char.ini
+anim/
+	deskslam.gif
+```
+This works with multiple subfolders as well - for example, `anim/young/damage`:
+```
+char.ini
+anim/
+	young/
+		damage.gif
+```
+  Consider using subfolders if your character has a lot of files.
+  
 
 ##### `<emote>` (idle and speaking animations)
 
+  
+
 Between the next #'s is the name of the actual animation when the character is idle and speaking, starting with `(a)` and `(b)` in the character folder, respectively. For instance, if your animation is named "cough", then the engine expects your idle animation to be named `(a)cough.gif` on the file system, and your talking animation to be named `(b)cough.gif`.
 
+  
+
 The engine will also scan for other supported file formats, such as `.apng` and `.webp`, as described above.
 
+  
+
 If no animations are found for an emote, the engine will fall back to static emotes. Static emotes don't use `(a)`/`(b)` prefixes - instead, an static emote named `cough` would be expected to be in the file `cough.png`.
 
+If you're the type who likes to organize, you can also place files in **subfolders**. To use subfolder mode, you simply have to prefix your emotes with a forward slash (`/`). For `(a)`/`(b)` type emotes in subfolder mode, create two folders named `(a)` and `(b)` and place your prefix-less files in the appropriate folders. For example, the emote `/angry` will search for a file named `angry.gif` in `(a)/` and `(b)/` for idle and talking respectively.
+
+As another example, the emote `/def/thinking` will search for a set of files arranged like this:
+```
+char.ini
+(a)/
+	def/
+		thinking.gif
+(b)/	
+	def/
+		thinking.gif
+```
+This is a popular way to declutter the root of your character folder.
+  
+
 ##### `<modifier>`
 
-Finally, there's a numbered modifier. The valid inputs here are: 0, 1, and 5.
+The modifier value controls pre-animations, sounds, and zooms. The valid inputs here are: 0, 1, 5, and 6.
+
+- `0`: Tells the client to not play the pre-animation or sound effect associated with the emote, Has no effect on `[FrameSFX]` (more on that later)
 
-- `0`: Tells the client to not play the pre-animation or sound effect associated with the emote unless 'Take that'/'Objection'/'Hold it' is used.
 - `1`: Plays the pre-animation and associated sound.
-- `5`: Zoom, in which the foreground desk or witness stand will not be displayed. Additionally, the background is replaced by speed lines.
+
+- `5`: Zoom, in which the foreground desk or witness stand will not be displayed. Additionally, the background is replaced by speed lines. Emotes with `5` will never play pre-animations.
+
+- `6`: Same as 5, except it will *always* play pre-animations.
+
+##### `<deskmod>` (optional)
+Introduced in release 2.6, this boolean value allows an emote to either force the desk/witness stand/overlay to be displayed, or force it to disappear. This takes precedence over all other factors affecting desk visibility.
```

Comment:
```suggestion
This option allows an emote to either force the desk/witness stand/overlay to be displayed, or force it to disappear. This takes precedence over all other factors affecting desk visibility.
```

Reply:
good point, probably making people google "boolean" is a bad idea